File: carrera-sdk/producer/go/src/carrera/addTxMonitorMessageBuilder.go
package carrera

import "carrera/CarreraProducer"

type AddTxMonitorMessageBuilder struct {
	addDelayMessageBuilder *AddDelayMessageBuilder
}

func NewAddTxMonitorMessageBuilder(addDelayMessageBuilder *AddDelayMessageBuilder) *AddTxMonitorMessageBuilder {
	return &AddTxMonitorMessageBuilder{addDelayMessageBuilder:addDelayMessageBuilder}
}

func (addTxMonitorMessageBuilder *AddTxMonitorMessageBuilder) Send() *CarreraProducer.DelayResult {
	return addTxMonitorMessageBuilder.addDelayMessageBuilder.Send()
}

File: carrera-sdk/producer/java/carrera-producer-sdk/src/main/java/com/xiaojukeji/carrera/producer/tx/AddTxMonitorMessageBuilder.java
package com.xiaojukeji.carrera.producer.tx;

import com.xiaojukeji.carrera.producer.AddDelayMessageBuilder;
import com.xiaojukeji.carrera.thrift.DelayResult;


public class AddTxMonitorMessageBuilder {

    private AddDelayMessageBuilder addDelayMessageBuilder;

    public AddTxMonitorMessageBuilder(AddDelayMessageBuilder addDelayMessageBuilder) {
        this.addDelayMessageBuilder = addDelayMessageBuilder;
    }

    public DelayResult send() {
        return this.addDelayMessageBuilder.send();
    }
}